src/main/java/com/example/java/test/junior/developer/security/SecurityConfig.java

```diff
@@ -0,0 +1,47 @@
+package com.example.java.test.junior.developer.security;
+
+import org.springframework.boot.autoconfigure.AutoConfiguration;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.security.config.annotation.method.configuration.EnableMethodSecurity;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityCustomizer;
+import org.springframework.security.web.SecurityFilterChain;
+import org.springframework.security.web.util.matcher.AntPathRequestMatcher;
+import org.springframework.security.web.util.matcher.RequestMatcher;
+
+@Configuration
+@AutoConfiguration
```

Comment:
Why do you use `@AutoConfiguration` here? Remember our main rule - don't write a single line you don't 100% understand the meaning of. This is not auto-configuration, just configuration (and you already have `@Configuration` annotation as well).